multus/templates/daemonSet.yaml

```diff
@@ -73,18 +73,22 @@ spec:
           mountPath: /host/etc/cni/net.d
         - name: cnibin
           mountPath: /host/opt/cni/bin
+{{- if .Values.manifests.configMap }}
```

Comment:
Please, add indentation to if-else statements to follow current style

Reply:
done!